prow/config.yaml

```diff
@@ -66,6 +66,7 @@ branch-protection:
         - cla/google
       required_pull_request_reviews:
         required_approving_review_count: 1
+        require_code_owner_reviews: true
```

Comment:
This seems like the only real change in here? It will be a lot easier to understand this change later if its the only thing in it.

Can you send the sorting PR separately?

Reply:
I also added the tools repo as mentioned in the PR description.